src/unittest/python/pmf_tests.py

```diff
@@ -91,6 +107,15 @@ def test_transform_results_in_predictions_on_transformed_values(self):
                                        'ANY_WIDTH')
         mock_log.assert_called_once_with('ANY_X_VALUE')
 
+    def test_deviance_test(self):
+        ps = pmf.PsychometricFunction(pmf.logistic, 0.5)
+        ps.params = np.array([1., 1., .03])
+        p, D = ps.deviance_test(self.data, nsamples=20)
+        self.assertSequenceEqual(D.shape, (20,))
+        self.assertIsInstance(p, float)
+        self.assertGreaterEqual(p, 0)
+        self.assertLessEqual(p, 1)
+
 
 class TestGrid(TestCase):
```

Comment:
The name "test_more_dims" does not make it obvious how it is different from "test_dims"

Reply:
Yes, but not part of the changes here, so that wouldn't be efficient to fix at this time.